File: docs/_docs/code-snippets/dotnet/OnHeapCaching.cs
using Apache.Ignite.Core.Cache.Configuration;

namespace dotnet_helloworld
{
    class OnHeapCaching
    {
        public static void Run()
        {
            // tag::onheap[]
            var cfg = new CacheConfiguration
            {
                Name = "myCache",
                OnheapCacheEnabled = true
            };
            // end::onheap[]
        }

    }
}

File: docs/_docs/code-snippets/java/src/main/java/org/apache/ignite/snippets/OnHeapCaching.java
package org.apache.ignite.snippets;

import org.apache.ignite.configuration.CacheConfiguration;

public class OnHeapCaching {

    public static void onHeapCacheExample() {
        //tag::onHeap[]
        CacheConfiguration cfg = new CacheConfiguration();
        cfg.setName("myCache");
        cfg.setOnheapCacheEnabled(true);
        //end::onHeap[]

    }
}
